### PR TITLE
Feature add release

### PR DESCRIPTION
**Pull request by markusranda**
- User can now choose which ProductVersion this Release is affiliated with.
- Fetches release notes from db.
- Now possible to create new release from web.
- Added isPublic switch.
- Added validation for all "fields".  This is without feedback..

### CODE REVIEW

File: src/requests/releaseNote.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export function getReleaseNotes() {
+  return Axios.get("releasenote", {
+    withCredentials: false,
+    headers: {
+      ["Access-Control-Request-Headers"]: "Content-Type",
+      ["Authorization"]: "Bearer " + getAuthToken()
+    }
+  });
+}
```
Comment (0us) [inline src/requests/releaseNote.js]: Plz bruk redux

Comment (markusranda) [reply src/requests/releaseNote.js]: Har du et konkret eksempel eg kan se på eller en artikkel?

Comment (0us) [reply src/requests/releaseNote.js]: Sjå LandingScreen, productActions og productReducer, og/eller den her https://www.youtube.com/watch?v=CVpUuw9XSjY&t=2074s

Comment (markusranda) [reply src/requests/releaseNote.js]: Lagt det inn som task å implementer Redux i løsninga på Azure Boards.

**Pull request merged**